NineAnimator/Controllers/Asset Manager/OfflineContentManager.swift

```diff
@@ -439,7 +439,14 @@ extension OfflineContentManager {
             }
             
             // Move the item
-            try fs.moveItem(at: location, to: destinationUrl)
+            do {
+                try fs.moveItem(at: location, to: destinationUrl)
+            } catch {
+                // Fallback that works on Jailbroken version
+                Log.info("[OfflineContentManager] Attempting to move the downloaded asset for task (%@) indirectly after failed attempt using FileManager: %@", downloadTask.taskIdentifier, error)
+                let data = try Data(contentsOf: location)
+                try data.write(to: destinationUrl)
+            }
```

Comment:
It would be better to use `fs.copyItem(at: location, to: destinationUrl)` if it doesn't triggers the code 513 error, since `Data(contentsOf: )` reads the entire file into memory.

```Swift
try fs.moveItem(at: location, to: destinationUrl)
try? fs.removeItem(at: location)
```

Reply:
Works. Seems there is no access to the temporary folder on jailbroken installs.